gtfu/src/main/java/gtfu/tools/AgencyYML.java

```diff
@@ -29,6 +34,10 @@ public AgencyYML(){
         processYML();
     }
 
+    /**
+    * Identify the agencyName and GTFS schedule URL for every agency represented in the file.
+    * It is straightforward to add additional fields to pull from the file.
+    */
```

Comment:
Documenting private fields is really going above and beyond. We don't need to go beyond documenting public fields.